src/changelog.ts

```diff
@@ -21,7 +56,11 @@ function stringifyCategory(category, changes) {
   return `${categoryHeader}\n${changeDescriptions}`;
 }
 
-function stringifyRelease(version, categories, { date, status } = {}) {
+function stringifyRelease(
+  version: string,
```

Comment:
This might be a more descriptive type to use here:
```suggestion
  version: Version | typeof unreleased,
```